data-access-analysis-tools/index-rsp-warnings.rst

```diff
@@ -12,98 +12,60 @@
 
-When in doubt, post questions in `the "Support -- Data Preview 0" category <https://community.lsst.org/c/support/dp0/49>`__ in the Community Forum. 
+When in doubt whether something is okay or to ask information about about planned future functionality, post questions in `the "Support -- Data Preview 0" category <https://community.lsst.org/c/support/dp0/49>`__ in the Community Forum.
 
+If you believe you have found a bug or wish to request a feature, please `open a Github Issue <https://github.com/rubin-dp0/Support/issues/new/choose>`_ instead.
 
-Data Sharing and Safeguards
----------------------------
-
-A number of safeguards for avoiding uptime or temporary data loss will not be present -- the resources are still in “trusted user” mode.
-
-The fact that anyone could PRUNE, there's no provenance for destruction, and how to avoid accidental wipe-outs.
-
-Do we have shared data sets? We did on the NCSA RSP. (HFC: what kinds of shared datasets are you thinking here? The only data set we officially provide is the DP0.1 data set. There are a lot more data on NCSA's filesystem and we never intended to have them on IDF.)
-
-Do we have access to other users’ home directories?
-
-Need some warnings about per-user data quota. 
-
-Need some warnings about shared-write area.
-
-
-
-RSP Compute Performance
------------------------
-
-Performance during DP0 may not reflect the performance of the final system (see also Q4), and the resources made available to DP0 delegates may not reflect the final user quotas of the operations-era RSP.
-
-How about running cron jobs? (HFC: I don't think we would let delegates run cron. But what is the motivation of wanting to run cron?)
-
-What about abusing resources?
+User data: No parachutes!
+-------------------------
 
-Testing how algorithms will scale: No access to user batch or parallelisation are available in DP0. Resources provided as part of DP0.1 are limited and meaningful scalability testing of algorithms is not practical. 
+When you access the RSP Notebook aspect, you have a home space you can use to store your notebooks, scripts, extra python modules and small files. Other users cannot see the contents of your home space, but you have no expectation of privacy from observatory staff monitoring the systems. If we experience an incident, we will attempt to restore the contents of your homespace. The contents of your homespace will not be deleted without attempting to notify you.
 
+Data in /scratch is not backed up.
 
+Data in /projects is available for use in the situation where you wish to share files with other RSP users. For this data preview, any files in you put in /projects are readable and writeable by all users. Take care you do not delete other people's files.
```

Comment:
In this wild-west butler they can easily share their butler files simply by telling their collaborator the run, dataset type and dataId they need to get at the file. I assume we'd rather people do that for IDF collaboration than telling people to download files from butler to put in the shared file system.

Reply:
Don't normal POSIX access controls with RSP-local UIDs/GIDs apply in `/project` (without "s", right?)?  Same as home directories.